assets/wizards/engagement/views/newsletters/index.js

```diff
@@ -148,40 +247,16 @@ const Newsletters = () => {
 				isOnboarding={ false }
 				onUpdate={ config => updateConfiguration( { newslettersConfig: config } ) }
 			/>
-			<SectionHeader
-				title={ __( 'Signup Forms', 'newspack' ) }
-				description={ () => (
-					<>
-						{ __(
-							'Connects your site to Mailchimp and sets up a Mailchimp block you can use to get new subscribers for your newsletter',
```

Comment:
Along with this removal, the Mailchimp-via-Jetpack connection in the Connections wizard should be removed, too:

<img width="786" alt="image" src="https://user-images.githubusercontent.com/7383192/177289330-e9b29d85-24dd-48f3-84a2-48172ce3b066.png">

Reply:
Done in ff382cb1cebd8f73461f9d204508ddb751937858